org.gnome.Books.json

```diff
@@ -54,13 +62,14 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/gnome-desktop/3.26/gnome-desktop-3.26.2.tar.xz",
-                    "sha256": "f7561a7a313fc474b2c390cd9696df1f5c1e1556080e43f4afe042b1060e5f2a"
+                    "url": "https://download.gnome.org/sources/gnome-desktop/3.30/gnome-desktop-3.30.1.tar.xz",
+                    "sha256": "1383703bedd71204e4a161fb163905410c8f651890f18b3fa8405506ba97789f"
                 }
             ]
         },
         {
             "name": "liboauth",
+            "config-opts": [ "--enable-nss" ],
```

Comment:
Reason for change?

Reply:
I took it from the upstream build instructions. It's because it doesn't build sucessfully against OpenSSL:
https://gitlab.gnome.org/GNOME/gnome-documents/commit/e5892de98ae58b2cfa631106128d04aeea32dbda